toolpath_planner:
  ros__parameters:
    moveit_planning_group: 'iiwa'
    tool_reference_frame: 'cutting_tool_base'
    end_effector_reference_frame: 'gripper_jaw_centre'
    part_reference_frame: 'implant'
    moveit_scale_velocity: 0.2
    moveit_scale_acceleration: 1.0
    moveit_planning_time: 30.0
    moveit_planning_attempts: 5
    desired_cartesian_velocity: 0.08
    desired_cartesian_acceleration: 0.2
    toolpath_height_offset: 0.0
    approach_offset: 0.03
    retreat_offset: 0.03
    retreat_height: 0.01
    debug_mode: true
    debug_wait_time: 500